### DISCUSSION TITLE
Accidentally ran `quarto publish` without `gh-pages`

### DISCUSSION THREAD

**Discussion by yinleon** [quarto-dev/quarto-cli#5951] [Q&A]
### Description

I made a dumb mistake and ran `quarto publish` without `gh-pages`, and now my website is just the README.md.

I have no idea how to fix this, has anyone else had this issue?

Thanks!

**Comment by mcanouil** [comment]
Could you explain with more details? What are you calling "website"?

`quarto render` is supposed to prompt you with a menu to select the target.
<img width="532" alt="screenshot of a terminal showing the output of 'quarto render' when no target is specified. The output is a menu with several options, such as GitHub Pages, Confluence, Quarto Pub, etc." src="https://github.com/quarto-dev/quarto-cli/assets/8896044/3e396a47-138d-4dfb-9448-540f5e77d7a8">
